.gitignore

```diff
@@ -1,6 +1,43 @@
+# Already present in Repo
 /target
 tarpaulin-report.html
 lcov.info
 *.tar.gz
 PKGBUILD
-tests/*.testcfg
\ No newline at end of file
+tests/*.testcfg
+#Added from https://www.toptal.com/developers/gitignore/api/rust,macos
+/.vscode
+### macOS ###
+.DS_Store
+.AppleDouble
+.LSOverride
+# Icon must end with two \r
+Icon
+# Thumbnails
+._*
+# Directories potentially created on remote AFP share
+.AppleDB
+.AppleDesktop
+Network Trash Folder
+Temporary Items
+.apdisk
+# iCloud generated files
+*.icloud
+
+### Rust ###
+# Generated by Cargo
+# will have compiled files and executables
+debug/
+target/
```

Comment:
These are already covered with the `/target` on line 2

Reply:
Thx - had just merged my default rust .gitignore.